libs/community/langchain_community/vectorstores/jaguar.py

```diff
@@ -0,0 +1,439 @@
+from __future__ import annotations
+
+import json
+import logging
+from typing import Any, List, Optional, Tuple
+
+from jaguardb_http_client.JaguarHttpClient import JaguarHttpClient
```

Comment:
this should be a conditional import

Reply:
The change has been made. It looks like now:

from typing import TYPE_CHECKING, Any, List, Optional, Tuple

if TYPE_CHECKING:
    from jaguardb_http_client.JaguarHttpClient import JaguarHttpClient

from langchain_core.documents import Document
from langchain_core.embeddings import Embeddings
from langchain_core.vectorstores import VectorStore